modules/loki-deployment-sizing.adoc

```diff
@@ -1,41 +1,23 @@
 // Module is included in the following assemblies:
-//cluster-logging-loki.adoc
+// * logging/cluster-logging-loki.adoc
+// * network_observability/installing-operators.adoc
 
 :_mod-docs-content-type: CONCEPT
 [id="loki-deployment-sizing_{context}"]
 = Deployment Sizing
 
 Sizing for Loki follows the format of `N<x>._<size>_` where the value `<N>` is number of instances and `<size>` specifies performance capabilities.
 
-[NOTE]
-====
-1x.extra-small is for demo purposes only, and is not supported.
-====
-
 .Loki Sizing
 [options="header"]
 |========================================================================================
-|                              | 1x.extra-small  | 1x.small            | 1x.medium
-| *Data transfer*              | Demo use only.  | 500GB/day           | 2TB/day
-| *Queries per second (QPS)*   | Demo use only.  | 25-50 QPS at 200ms  | 25-75 QPS at 200ms
-| *Replication factor*         | None            | 2                   | 3
-| *Total CPU requests*         | 5 vCPUs         | 36 vCPUs            | 54 vCPUs
-| *Total Memory requests*      | 7.5Gi           | 63Gi                | 139Gi
-| *Total Disk requests*        | 150Gi           | 300Gi               | 450Gi
+|                                          | 1x.extra-small     | 1x.small            | 1x.medium
+| *Data transfer*                          | 100GB/day          | 500GB/day           | 2TB/day
+| *Queries per second (QPS)*               | 1-25 QPS at 200ms  | 25-50 QPS at 200ms  | 25-75 QPS at 200ms
+| *Replication factor*                     | None               | 2                   | 3
+| *Total CPU requests*                     | 14 vCPUs           | 36 vCPUs            | 54 vCPUs
+| *Total CPU requests incl. Ruler*         | 16 vCPUs           | 42 vCPUs            | 70 vCPUs
+| *Total Memory requests*                  | 39Gi               | 63Gi                | 109Gi
+| *Total Memory requests incl. Ruler*      | 43Gi               | 83Gi                | 141Gi
+| *Total Disk requests*                    | 150Gi              | 300Gi               | 450Gi
```

Comment:
Does it need to explain what Ruler is?

If LokiStack is being used for Network Observability, then Ruler isn't being used.

Reply:
Added conditionals to the module so the ruler part should only show up when this module is included in the Logging docs now, PTAL at the previews 🙂

Reply:
/lgtm